src/node/handler/PadMessageHandler.js

```diff
@@ -951,6 +949,9 @@ const handleClientReady = async (socket, message, authorID) => {
       padIds.readonly || !webaccess.userCanModify(message.padId, socket.client.request);
 
   const {session: {user} = {}} = socket.client.request;
+  const {accessStatus} = await securityManager
+      .checkAccess(sessioninfos[socket.id].auth.padID, sessioninfos[socket.id].auth.sessionID, sessioninfos[socket.id].auth.token, user);
+
```

Comment:
Can you explain why this is needed here?

Reply:
It return to the previous form.